packages/cactus-common/src/main/typescript/verifier/jwt-message-authentication.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020-2022 Hyperledger Cactus Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import crypto from "crypto";
+import jwt from "jsonwebtoken";
```

Comment:
@outSH Could you either make this cross-platform by any chance? (the common package is designed to be useable in both the browser and NodeJS environments). Alternatively it could go in another package too, I'm flexible on options, but want to avoid exporting functions to the browser that are known to not work.

I'm not sure if this is actually the diff of another PR that this one depends on or if this is the one I should be reviewing. If this is coming from another PR then this PR is LGTM (and I'll ask you to fix it in that other PR instead), just let me know which way it goes.

Reply:
@petermetz Yeah you already made this comment here - https://github.com/hyperledger/cactus/pull/2047

Reply:
This is fixed

Reply:
@outSH Oh, that's why it seemed so familiar and redundant! /facepalm sorry!